recipes/soappy/meta.yaml

```diff
@@ -0,0 +1,64 @@
+{% set name = "SOAPpy" %}
+{% set version = "0.12.22" %}
+{% set sha256 = "e70845906bb625144ae6a8df4534d66d84431ff8e21835d7b401ec6d8eb447a5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.zip
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.zip
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [ not py2k]
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - wstools
+    - defusedxml
+
+test:
+  # Some package might need a `test/commands` key to check CLI.
+  # List all the packages/modules that `run_test.py` imports.
+  imports:
+    - SOAPpy
+    - SOAPpy.Client
+    - SOAPpy.Config
+    - SOAPpy.Errors
+    - SOAPpy.NS
+    - SOAPpy.Parser
+    - SOAPpy.Server
+    - SOAPpy.SOAPBuilder
+    - SOAPpy.SOAP
+    - SOAPpy.Types
+    - SOAPpy.URLopener
+    - SOAPpy.Utilities
+    - SOAPpy.version
+    - SOAPpy.WSDL
+
+about:
+  home: https://github.com/kiorky/SOAPpy
+  license: BSD-3-Clause
+  license_family: BSD
+  license_file: '{{ RECIPE_DIR }}/LICENSE'
+  summary: 'Simple to use SOAP library for Python'
+
+  description: |
+    Disclaimer
+    ==========
+    Please use `suds <https://pypi.python.org/pypi/suds>`_ rather than SOAPpy.
+    SOAPpy is old and clamsy.
+  dev_url: https://github.com/kiorky/SOAPpy
+
+extra:
+  recipe-maintainers:
+    - pvanheus
+    - AjitPS
```

Comment:
@AjitPS -- are you ok being added as a co-maintainer for the `SOAPpy` and `wstools` recipes on conda-forge? You were added by @pvanheus.